packages/cmcd-validator-demo-app/README.md

```diff
@@ -6,9 +6,43 @@
 
 # cmcd-react-test-app
 
-The cmcd-react-test-app allows you to run an instance of shaka player and see CMCD messages on a display text box.
+##  Table  of  contents
 
-## Before start
+- [cmcd-react-test-app](#cmcd-react-test-app)
+	- [Table  of  contents](#table--of--contents)
+	- [Overview](#overview)
+	- [Usage](#usage)
+	- [Setup](#setup)
+		- [Before start](#before-start)
+		- [Run local](#run-local)
+	- [Troubleshooting](#troubleshooting)
+  
+## Overview
+
+This React app is designed to validate CMCD (Common Media Client Data) requests from different players. It provides a user-friendly interface to enter a manifest URL and view the requests made by the player, along with their validation status.
+
+
+## Usage
+To validate CMCD requests using this app, follow these steps:
+
+1. Select a player.
+   
+![](images/select-player.png)
+
+2. Enter the manifest URL and click "Play" to start playback.
+   
+![](images/content-playing.png)
+> **_NOTE:_** The requests made by the player will be displayed in the panel on the right-hand side. The color of the request indicates its validation status: green for valid, red for invalid, and yellow for warnings.
+
+3. To view the validation output for a request, click on it and scroll down.
+   
+![](images/validation-datail.png)
+
+
+
+## Setup 
+
+### Before start
 
 Make sure of run the install and build of npm before run the local server:
```

Comment:
Make sure *to run / of running* the install and build of npm before running the local server